x-pack/plugins/fleet/server/services/secrets.test.ts

```diff
@@ -567,3 +567,183 @@ describe('getPolicySecretPaths', () => {
     });
   });
 });
+
+describe('diffSecretPaths', () => {
```

Comment:
Could we add a testcase for a field that was not set in old version, and has a new secret value in new version?